### PR TITLE
Added timeout to fix android 10 crash issues

### PR DESCRIPTION
**Pull request by Krakaw**
### References Issues:
https://github.com/facebook/react-native/issues/27204#issuecomment-590186372
### Environment Variables:
 * No change

### CODE REVIEW

File: src/Auth/Login.tsx [hunk 1]
```diff
@@ -50,12 +50,13 @@ async function getCredentials(): Promise<Maybe<Credentials>> {
...
 
 const LogIn = ({
-  loggingIn,
-  loggingInWithFacebook,
-  navigation,
-}: LoginProps) => {
+                 loggingIn,
+                 loggingInWithFacebook,
+                 navigation,
+               }: LoginProps) => {
```
Comment (coka) [inline src/Auth/Login.tsx]: The linter you are using seems to be conflicting with Prettier.

File: src/Auth/Login.tsx [hunk 2]
```diff
@@ -72,6 +73,13 @@ const LogIn = ({
     })
   }, [])
 
+  useEffect(() => {
+    setTimeout(() => {
+      setEditable(true)
+    }, 100)
+  }, [])
```
Comment (coka) [inline src/Auth/Login.tsx]: I would add a task to the backlog for limiting this effect to Android with `Platform` (or, better yet, to Xiaomi devices with something like [react-native-device-info](https://github.com/react-native-community/react-native-device-info#getbrand).

**Comment by Krakaw** [comment]
@coka Should we be adding this hack to all email input fields throughout the app?

**Comment by coka** [comment]
@Krakaw Yeah, I think so, if it's happening in screens unrelated to `SecureStore`. We can probably do something along the lines of:
```javascript
const withXiaomiHack = (EmailInput) => (props) => {
  const [editable, setEditable] = useState(Platform.OS !== 'android')
  useEffect(() => {
    if (Platform.OS === 'android') {
      setTimeout(() => {
        setEditable(true)
      }, 100)
    }
  }, [])

  return <EmailInput editable={editable} {...props} />
}
```

**Comment by saketkumar** [comment]
@Krakaw Is this fix solved the issue for crash ?

**Comment by Krakaw** [comment]
@saketkumar we used the solution that @coka proposed [here](https://github.com/big-neon/bn-mobile-react/pull/315#issuecomment-596988872)

**Comment by saketkumar** [comment]
@Krakaw So Is this fixed the crash issue ?

**Comment by Krakaw** [comment]
> @Krakaw So Is this fixed the crash issue ?
> 

Yes

**Pull request merged**